src/app/shared/components/basket/basket-approval-info/basket-approval-info.component.html

```diff
@@ -0,0 +1,46 @@
+<div *ngIf="approval?.approvalRequired" class="row">
+  <div class="col-12">
+    <div class="text-right">
+      <div class="approval-required" title="{{ 'approval.cart.approver' | translate }}">
+        <span>{{ 'approval.cart.approval_required' | translate }}</span>
+        <!-- details link and dialog -->
+        <ish-modal-dialog-link
+          linkText="approval.cart.link.details"
+          [options]="{ titleText: 'approval.details.heading' | translate }"
+          [ngClass]="'details-link'"
```

Comment:
There is no need for `[ngClass]="'details-link'"` if the CSS class is just a simple string value.